torch/_inductor/codegen/triton.py

```diff
@@ -2741,8 +2766,8 @@ def codegen_cooperative_reduction_peer_combine(self, result_var, dtype):
             strip=True,
         )
         self.post_loop_store.writeline(
-            f"{result_var}_peers = tl.load({result_var}_ws + (xindex * RSPLIT + tl.arange(0, RSPLIT){expand}), "
-            f"{mask}, eviction_policy='evict_first')"
+            f"{result_var}_peers = tl.load({result_var}_ws + (xindex * RSPLIT + rsplit_arange), "
+            f"rsplit_mask, eviction_policy='evict_first', other=triton_helpers.zero_other({result_var}_ws, rsplit_mask))"
```

Comment:
Wouldn't the other mask depend on the type of reduction ? or is there something missing

Reply:
The mask will be `None` if RSPLIT is a power of 2.

I believe `other=0` works for all reduction types, is there one that requires something else?

Reply:
min

Reply:
Ah right, I'll plumb that through.